bin/download_anonymised

```diff
@@ -2,13 +2,21 @@
 set -e
 
 LAA_REF=$1
+DESTINATION_ENVIRONMENT=$2
 
-echo "Downloading anonymised record..."
+echo "Downloading anonymised record from production..."
 OUTPUT=$(kubectl exec deploy/laa-crime-application-store-app \
 -it -n laa-crime-application-store-production -- \
 bundle exec rails anonymised:download\[$LAA_REF\])
 
-echo "Importing anonymised record..."
-ANONYMISED_DOWNLOAD_OUTPUT=$OUTPUT bundle exec rails anonymised:import\[ANONYMISED_DOWNLOAD_OUTPUT\]
+echo "Importing anonymised record to $2..."
+
+if [[ $DESTINATION_ENVIRONMENT == "local" ]]; then
+  ANONYMISED_DOWNLOAD_OUTPUT=$OUTPUT bundle exec rails anonymised:import\[ANONYMISED_DOWNLOAD_OUTPUT\]
+else
+  kubectl exec deploy/laa-crime-application-store-app \
+  -it -n laa-crime-application-store-$DESTINATION_ENVIRONMENT -- \
+  env ANONYMISED_DOWNLOAD_OUTPUT="${OUTPUT}" bundle exec rails anonymised:import\[ANONYMISED_DOWNLOAD_OUTPUT\]
```

Comment:
Same here, `env` isn't needed but the shell escape quoting should be

Reply:
FWIW env really is needed - without it you get `error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec...`. You may not have spotted that line 19 is not a standalone statement, it's a continuation of the statement starting on line 17, and is therefore  run in the kube pod and needs slightly special escaping to get it there.

Reply:
Yeah line break again, it is. Line 15 should have the same shell escape quoting though